ajax/libs/dplayer/package.json

```diff
@@ -14,23 +14,14 @@
     "html5"
   ],
   "author": "DIYgod",
-  "license": {
-    "type": "SATA",
-    "url": "https://github.com/DIYgod/DPlayer/blob/master/LICENSE"
-  },
+  "license": "MIT",
   "npmName": "dplayer",
   "npmFileMap": [
     {
       "basePath": "dist",
       "files": [
         "**/*"
       ]
```

Comment:
Please add origin `npmFileMap` back, thanks. (The origin as the following)
```js
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    },
    {
      "basePath": "src",
      "files": [
        "DPlayer.!(scss)"
      ]
    }
  ]
```
Thanks.

Reply:
@sufuf3 Why? Do we need `src`?

Reply:
@maruilian11 Don't we need non-minify files?
As you can see, the lib's files under `dist` folder only add minified files. https://github.com/MoePlayer/DPlayer/tree/v1.16.0/dist
The non-minify files is under `src`
Thanks

Reply:
@sufuf3 There is no `src` now in npm package.

Reply:
ping @sufuf3. Thanks.

Reply:
OK.
I think it's fine.
But I think we can host non-minified files.
Maybe using git auto-update is good. 
What do you think?

@PeterDaveHello May I ask about auto-update config?
Because the current auto-update is npm auto-update. But I think we can add non-minified files from GitHub.
Not sure which one is good to be used.
Could you help us?

Thanks.